core/tests/Test.Microsoft.Identity.Core.UIAutomation/XamarinUITestController.cs

```diff
@@ -31,9 +31,17 @@
 using System.Linq;
 using Xamarin.UITest;
 using System.Globalization;
+using Xamarin.UITest.Queries;
 
 namespace Test.Microsoft.Identity.Core.UIAutomation
 {
+    public enum XamarinSelector
+    {
+        ByAutomationId,
+        ByHtmlIdAttribute,
+        ByHtmlValue
```

Comment:
This was the more difficult issue to overcome - when doing a "SELECT_ACCOUNT" the web ui has almost no IDs. I found a way to select the web element by the name of the user.